### DISCUSSION TITLE
FORCE Coregistration / base-images: Risk of getting 'false' tie-points by areas of intermediate stability (floodplain)?

### DISCUSSION THREAD

**Discussion by Jay-Sig** [davidfrantz/force#112] [Q&A] [answered]
Hello,

I'm about to create a Sentinel time series and for that reason to coregister some Sentinel-images. So, I am looking for the base-image setup which is most suitable to my project properties.

First some key points of the project:
- Area of interest is the floodplain (furcation) section of an alpine river, width of the floodplain between a few 100 up to 1000 meters. 
- The extent of the AOI is about 30 km x 90 km (lon/lat), intersecting with four (2x2) Sentinel footprints. 
- One half of the section is situated in the mountains, the other half in the following lowland area.  
- Tiling: 6 x 7 tiles, each 36 km (to cover all Sentinel footprints). The AOI lies within three of these tiles
- Base topic are floodplain dynamics, specifically the positonal changes of the river channels. 
- Period of interest is 2018 to 2020. 
- Total period for the base-images is intentionally from 2015 or 2016 to 2020 (due to tutorial-recommended minimum period of five years)

Now into my issue:
Over the years the course of the river channels fundamentally changes with each stronger flood, mostly once or twice a year. These changes also concern the positioning of vegetation and gravel banks within the floodplain and its borders. **But**, depending on the year-specific occurence of the single flood events the position of channels, vegetation and gravel banks can be stable for several months as well (often late spring to early autumn), sometimes even for a whole year. 
Because of this intermediate stability I am now 'a little bit afraid' of getting 'false' tie-points within the floodplain area, leading to unsufficient coreg-results. This concern is still enhanced because, with NIR, river channel surfaces seem to differentiate quite clearly from many other surfaces, and thus might be particularely suitable for calculating tie points.
Therefore at the moment, I tend to use one single base-image covering the total 5-year-period instead of three 2-years-base-images. By using a longer in-one-base-image-period with subsequently containing more alteration in the floodplain, I hope 'to trigger the algorithm' not to look in the floodplain area for ostensible or 'false' stable points, respectively.

My questions:
1) Is this a reasonable thought? Or doesn't it play any role (e.g. because water surfaces do not matter to tie-point-detecting, or any other reason)?
2) What could be the advantage of using three 2-years-base-images?
And byond my specific issue: 
3) As the tutorial generally recommends better to take at least 5 base-image years: Could 4 years be enough in my case? 
4) Would you see any problems, if reducing the tile size down to 18 or even 15 km, in context with the coregistration?
5) May you have any other advices on coregistration in my specific case, concerning e.g. used months (<-- winter/snow/shadow/mountains), interpolation options, ...?

Thanks in advance for any answer

**Comment by davidfrantz** [comment, marked as answer]
Hi @Jay-Sig 

these are all valid points and I cannot fully answer this. You will need to experiment a bit and evaluate the results visually. 

In general, land cover change (like a changing river) can be problematic, of course. In order to reduce this possibility, you can use fewer years for generating the base images (if there is enough data to fill the base image).

However, there are some safeguards in the algorithm that may help you, thus I assume that the results will be OK.
- Water is generally excluded from being a tie point. 
- We only use an affine transformation, which is computed from all tie points in the image, thus incorrect tie points in the relatively small river area might be averaged out. However, you have to make sure that your base image covers the complete Sentinel-2 images that you process (not just the AOI).

The tile size does not influence the coregistration.

Cheers,
David

**Reply by Jay-Sig** [reply]
Thanks for your response, David.
There is just one non-plausible aspect for me:
You suggested to "use fewer years for generating the base images" in order to avoid or reduce possible 'false'-points.

Now, I imagine the algorithm somehow this way: It looks for pixels or small pixel groups, which deliver (near) the same signals for all monthly NIR-Images (or at least for a maximum number) at the same location and over the whole period. Thus, the more images deliver the same signal at a given point, the more this point can be considered to be stable and subsequently be a tie-point. 

So, for me that would mean: The longer the period, the more change will happen within the floodplain, the less the algorithm is 'seduced' to calculate tie-points from the river area and instead is forced to build its tie-points from the (more stable) surroundings. 
Shortly: Longer periods lead to lower risk of getting false tie points in the floodplain or to exclusion of the floodplain, respectively. Quite the opposite of your proposal, ...?

**Reply by davidfrantz** [reply]
Hi @Jay-Sig ,

the matching is basically based on contrast. If contrast of one patch is similar between the base image and the target image, we have a high chance of finding a tie point. 

Thus, if the river changes its course, let's say 30m in one direction, we may get a false tie point. 

If we use a 5 year aggregation period, and if this change is permanent, and if it happens approximately in the middle of this time period, and we assume that data availability is similar before and after the change (many assumptions here), we may get a ghosting effect in the average base image. This could in theory result in an incorrect tie point. If you use shorter time periods, the chance that this happens is lower, and the chance is higher that the image to be registered matches better with the circumstances present during the period of the base image.

However, as we do not use polynomials or a spline transformation, other tie points in stable areas will correct for this, and this point will likely be classified as outlier, and will not be used.
